src/modules/Popup/Popup.js

```diff
@@ -375,8 +372,7 @@ Popup.propTypes = {
 
 Popup.defaultProps = {
   disabled: false,
-  eventsEnabled: true,
```

Comment:
As `eventListeners` is available via `modifier` I decided to keep prop to avoid an additional breaking change